copier.yml

```diff
@@ -38,41 +38,36 @@ use_ingestion:
 
 ingestion_framework:
     type: str
-    when: "{% if use_ingestion %}true{% endif %}"
     help: What ingestion framework would you like to use?
     choices:
         Airbyte: airbyte
     default: airbyte
```

Comment:
Let's remove this. We will have only one framework for know

Reply:
Even with default? So we don't want to show future possibilities? I guess that's fine, we don't need to bother users when there is no choice anyway.